### ISSUE TITLE
Unable to reach the accuracy reported in the paper

### ISSUE THREAD

**Issue by HantongXING** [TorchDSP/torchsig#133]
We generated our own WBSig53 dataset and downloaded the DETR based model and parameters for evaluation. On the impaired test set with 25000 signals, the mAP of DETR-B4-NANO was 61.30, and the mAP of DETR-B4-NANO-family was 69.69, which are much lower than what was reported in the paper. Other DETR based models also have similar results.

I want to know if this is caused by the randomness of the generated dataset? Is there a solution?

**Comment by gvanhoy** [comment]
These results were produced with the wideband dataset available in release ```v0.2.0```, which should generate a random-but-seeded version of the dataset that's the same as what was trained on. Did you use that version?

**Comment by HantongXING** [comment]
> No, we are using v0.3.0 actually. The results based on DETR are very excellent, but it seems that the published parameters are overfitting a specific test set. This may be due to insufficient sample richness in the test set.

> These results were produced with the wideband dataset available in release `v0.2.0`, which should generate a random-but-seeded version of the dataset that's the same as what was trained on. Did you use that version?

**Comment by HantongXING** [comment]
No, we are using v0.3.0 actually. The results based on DETR are very excellent, but it seems that the published parameters are overfitting a specific test set. This may be due to insufficient sample richness in the test set.

**Comment by gvanhoy** [comment]
Some of the changes from v0.3.0 included increasing the realism in how some signals were generated. Because the distribution in both training and validation sets changed, the drop in accuracy may be coming from using a model that is pre-trained on v0.2.0 and then validated on v0.3.0, which might make it look like there's overfitting because some signals were unchanged across this verison number.

If you do a per-class error analysis, I would expect that the FSK's would fail the most in your experiments because these changed the most. 

We're working towards a better naming/versioning system of the dataset here. Since the datasets are entirely synthetic, there is room to improve and adjust the dataset unlike other fixed datasets.

If you'd like to reproduce results in the paper, I think it's best to use ```v0.2.0.```
